composer.json

```diff
@@ -19,10 +19,14 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7 || ^6.0"
+        "elife/api-problem": "^1.0",
```

Comment:
a bit strange that a logging library also brings in an API library (you could log without having an API); as long as it's not circular, not an actual problem

Reply:
I would make `ApiProblemException` implement `HttpExceptionInterface`, but it comes from Symfony's HttpKernel. (Unlikely we would ever not use that, but it's an extract dependency...)